pkg/reconciler/openshift/tektonhub/extension.go

```diff
@@ -196,3 +201,64 @@ func getRouteHost(manifest *mf.Manifest, routeName string) (string, error) {
 	}
 	return hostUrl, nil
 }
+
+func UpdateDbDeployment() mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "Deployment" {
+			return nil
+		}
+
+		d := &appsv1.Deployment{}
+		err := runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, d)
+		if err != nil {
+			return err
+		}
+
+		if d.Name == "db" {
+			env := d.Spec.Template.Spec.Containers[0].Env
+
+			replaceEnv(env)
+
+			d.Spec.Template.Spec.Containers[0].Env = env
+
+			mountPath := "/var/lib/pgsql/data"
+			d.Spec.Template.Spec.Containers[0].VolumeMounts[0].MountPath = mountPath
+
+			replaceProbeCommand(d.Spec.Template.Spec.Containers[0].ReadinessProbe.Exec.Command)
+			replaceProbeCommand(d.Spec.Template.Spec.Containers[0].LivenessProbe.Exec.Command)
+
+			unstrObj, err := runtime.DefaultUnstructuredConverter.ToUnstructured(d)
+			if err != nil {
+				return err
+			}
+			u.SetUnstructuredContent(unstrObj)
+
+			return nil
+		}
+
+		return nil
+	}
+}
+
+func replaceProbeCommand(data []string) {
+	if strings.Contains(data[2], "POSTGRES_USER") {
+		data[2] = strings.ReplaceAll(data[2], "POSTGRES_USER", "POSTGRESQL_USER")
+	}
+	if strings.Contains(data[2], "POSTGRES_DB") {
+		data[2] = strings.ReplaceAll(data[2], "POSTGRES_DB", "POSTGRESQL_DATABASE")
+	}
+}
+
+func replaceEnv(envs []corev1.EnvVar) {
+	for i, e := range envs {
+		if e.Name == "POSTGRES_DB" {
+			envs[i].Name = strings.ReplaceAll(e.Name, "POSTGRES_DB", "POSTGRESQL_DATABASE")
+		}
+		if e.Name == "POSTGRES_USER" {
+			envs[i].Name = strings.ReplaceAll(e.Name, "POSTGRES_USER", "POSTGRESQL_USER")
+		}
+		if e.Name == "POSTGRES_PASSWORD" {
+			envs[i].Name = strings.ReplaceAll(e.Name, "POSTGRES_PASSWORD", "POSTGRESQL_PASSWORD")
```

Comment:
this could be simplied as as
```
_, ok := map[e.name]
if ok {
envs[i].Name = map[e.name]
}
```
and the map would be
```
replaceVal := map[string]string {
"POSTGRES_DB": "POSTGRESQL_DATABASE"
}
```